config/dictionaries/resource.json

```
@@ -535,8 +535,8 @@
         "description": "",
         "method": "PATCH",
         "endpoint": "/v6/device_tag",
-        "filters": "?\\$filter=device/uuid%20eq%20'<UUID>'",
-        "data": "{\n    \"tag_key\": \"<EXISTING-KEY>\",\n    \"value\": \"<NEW-VALUE>\"\n}"
+        "filters": "?\\$filter=(tag_key%20eq%20'<EXISTING-KEY>')%20eq%20(device/uuid%20eq%20'<DEVICE_ID>')",
```

Do you mean v ?
```suggestion
        "filters": "?\\$filter=(tag_key%20eq%20'<EXISTING-KEY>')%20eq%20(device%20eq%20<DEVICE_ID>)",
```

Which made me spot the `create-tag` right above, which afaik shouldn't use quotes around the `<DEVICE_ID>` there either...

---

In fact, it's one of these two options:

- DEVICE_ID without quotes
- DEVICE_UUID with quotes

I'll submit the change with the DEVICE_ID option as it seems to be the preferred one